msteams-platform/bots/how-to/authentication/bot-sso-register-aad.md

```diff
@@ -157,7 +157,7 @@ You need:
     > [!IMPORTANT]
     >
     > - **Standalone bot**: If you're building a standalone bot, enter the application ID URI as api://botid-{YourBotId}. Here, {YourBotId} is your Azure AD application ID.
-    > - **Application ID URI for app with multiple capabilities**: If you're building an app with a bot, a messaging extension, and a tab, enter the application ID URI as `api://fully-qualified-domain-name.com/BotId-{YourClientId}`, where the BotId is your bot app ID.
+    > - **Application ID URI for app with multiple capabilities**: If you're building an app with a bot, a messaging extension, and a tab, enter the application ID URI as `api://fully-qualified-domain-name.com/botId-{YourClientId}`, where `{YourClientId}` is your bot app ID.
```

Comment:
After reading so many docs, I believe `botId` should be `botid`, though I don't know whether `botId` could work as well.

Reply:
Hi @yuezk ,

We've updated `botId` to `botid`.